polling_stations/apps/data_importers/management/commands/import_chorley.py

```diff
@@ -28,13 +20,13 @@ def address_record_to_dict(self, record):
         ]:
             rec["accept_suggestion"] = True
 
-        # just bin the UPRNs on these ones
-        if uprn in [
-            "100010391254",  # PR75TW -> L402QN : 41 NEW STREET, ECCLESTON
-            "100010391256",  # PR75TW -> L402QN : 43 NEW STREET, ECCLESTON
-            "10070371313",  # L402QN -> PR75TW : 41 NEW STREET, MAWDESLEY, ORMSKIRK
-            "10070371314",  # L402QN -> PR75TW : 43 NEW STREET, MAWDESLEY, ORMSKIRK
-        ]:
-            rec["accept_suggestion"] = False
-
         return rec
+
+    def station_record_to_dict(self, record):
+        if (
+            record.pollingstationname == "ST JOHN THE DIVINE CHURCH HALL"
+            and record.pollingstationaddress_2 == "COPPULL"
+        ):
+            # missing post code
+            record = record._replace(pollingstationpostcode="PR7 5AB")
```

Comment:
I think I noticed it because it looked to be way out of area, but I'll check again. I got the postcode from the Google info box, so in future I should probably be less trusting of whatever magic they do.

Reply:
Cool :). TBH I just look in addressbase (or royal main website, or google (probably in that order)) for a suggestion and then fire it over to the council (via trello board) for confirmation - they're so much better placed to evaluate what it should be.

Reply:
Yep; looks like you had the right postcode. Now fixed, with a reference to AddressBase as source.

Reply:
The Council agrees with you :+1:  - bet they just googled it though :grinning:

Reply:
Went with 5AB :-)